scanpy/tools/_umap.py

```diff
@@ -118,9 +118,10 @@ def umap(
         init_coords = init_pos
     from sklearn.utils import check_random_state
     random_state = check_random_state(random_state)
-    n_epochs = maxiter
+    n_epochs = 0 if maxiter is None else maxiter
```

Comment:
0 epochs will result in no embedding. But `None` is not legit anymore?

Reply:
It was changed, now 0 is the same as None before.

Reply:
https://github.com/lmcinnes/umap/blob/a858c6322a3e682d8daf9c17e13ac023f3e18cfa/umap/umap_.py#L921

Reply:
Thanks!